playbooks/files/plugins/maas_common.py

```diff
@@ -43,7 +43,7 @@
 # v2 keystone auth is still necessary until RPCR switches over to v3 auth
 
 
-OPENRC = '/root/openrc-maas'
+OPENRC = '/root/openrc'
```

Comment:
This also should probably be settable, though it's able to be overridden if called correctly in the get_auth_info method.